src/controllers/message.js

```diff
@@ -29,7 +29,18 @@ const getAllMessages = async (req, res) => {
     const { userId } = req.user
     const messages = await Message.find({
         $or: [{ postedByUser: userId }, { receivedByUser: userId }], //We find all those messages which our userId sent or received. .
-    }).sort({ createdAt: 1 })
+    })
+        .populate({
+            path: 'postedByUser',
+            select: 'username -_id',
+            model: 'User',
+        })
+        .populate({
+            path: 'receivedByUser',
+            select: 'username -_id',
```

Comment:
I think just select: 'username id' works here

Reply:
so, does it mean we don't want to remove the Id from receivedByUser and postedByUser right? because  select: 'username id' returns`"{ _id: new ObjectId(\"641a9ac03cf5c88e0ce54c35\"), username: 'John' }":`